Packages/com.unity.inputsystem/InputSystem/Editor/ProjectWideActions/ProjectWideActionsAsset.cs

```diff
@@ -15,15 +16,9 @@ internal static class ProjectWideActionsAsset
 
         internal static class ProjectSettingsProjectWideActionsAssetConverter
         {
-            internal const string kAssetPath = "ProjectSettings/InputManager.asset";
+            internal const string kAssetPathInputManager = "ProjectSettings/InputManager.asset";
             internal const string kAssetName = InputSystem.kProjectWideActionsAssetName;
```

Comment:
Could name this kAssetNameProjectWide but not critical

Reply:
Not sure why defining a constant set to another constant? However...
InputSystem.kProjectWideActionsAssetName should no longer exist, so rather move that constant here instead. Its only relevant for the previous solution. It seems to not be used anywhere else....

Reply:
That's probably a result of starting in trunk and then moving to your branch - will define the constant here and remove the old one in InputSystem.cs

Reply:
Sounds good.